matter_server/server/device_controller.py

```diff
@@ -56,13 +72,14 @@ def __init__(
         self.compressed_fabric_id: int | None = None
         self._interview_task: asyncio.Task | None = None
 
-    @property
-    def fabric_id(self) -> int:
-        """Return Fabric ID."""
-        return cast(int, self.chip_controller.fabricId)
-
     async def initialize(self) -> None:
         """Async initialize of controller."""
+        # (re)fetch all PAA certificates once at startup
+        # NOTE: this must be done before initializing the controller
+        asyncio.create_task(fetch_certificates(PAA_ROOT_CERTS_DIR))
```

Comment:
Shouldn't we await fetching the certificates?

Reply:
oops, yes, copy paste error